### DISCUSSION TITLE
Is 1.2.0 gonna be released soon?

### DISCUSSION THREAD

**Discussion by GCSBOSS** [rustdesk/rustdesk#4516] [General]
Seems like nightly is going on for 8 months or something. Is the next version coming out soon? Is there some milestone set to be achieved before it can be released?

**Comment by Root-Core** [comment]
All I can find about it is this: https://github.com/rustdesk/rustdesk/issues/3819#issuecomment-1535900177 (Which is resolved, I guess)
Also there was already a discussion: https://github.com/rustdesk/rustdesk/discussions/3319

I would really love RustDesk to be released on Flathub (at least as beta), but I guess we have to wait a bit longer.
The 1.2.0 release will be huge! It has improved so much.. but a release means more user and more effort for the devs.

**Comment by rustdesk** [comment]
Yes, we will release it in this month (June), we can not wait either.

**Reply by Aqua1ung** [reply]
I am guessing that Wayland support is (or will be) out of beta, right?

**Reply by falynx** [reply]
I really hope this does happen this month (only 3 days left), because I've been *dying* to delete Anydesk, but I still can't because of a few nagging issues with Rustdesk on iOS/ipadOS. Can we get some confirmation that this is still on track for a release this month (June 2023)?

**Comment by Tarek-Hasan** [comment]
@rustdesk As next version going to be complete rewrite of the app, you should release it as version 2.0.0

**Comment by egpgit** [comment]
Is there any news about final version of 1.2.0? June is ends tomorrow and I haven't seen news about it.

I would like to know it because I'm waiting to deploy the client to multiple computers

Thanks a lot for your work
